src/pages/pass-finalisation/cat-c/pass-finalisation.cat-c.page.ts

```diff
@@ -0,0 +1,263 @@
+  getGearboxCategory,
+  isAutomatic,
+  isManual,
+} from '../../../modules/tests/vehicle-details/vehicle-details.selector';
+import { GearboxCategoryChanged } from '../../../modules/tests/vehicle-details/vehicle-details.actions';
+import { CAT_C } from '../../page-names.constants';
+import { getTestSummary } from '../../../modules/tests/test-summary/test-summary.reducer';
+import { isDebriefWitnessed, getD255 } from '../../../modules/tests/test-summary/test-summary.selector';
+import {
+  D255Yes,
+  D255No,
+  DebriefWitnessed,
+  DebriefUnwitnessed,
+} from '../../../modules/tests/test-summary/test-summary.actions';
+import { OutcomeBehaviourMapProvider } from '../../../providers/outcome-behaviour-map/outcome-behaviour-map';
+// TODO: MES-4287 Import Cat C behaviour map
+import { behaviourMap } from '../../office/office-behaviour-map.cat-be';
+import { ActivityCodes } from '../../../shared/models/activity-codes';
+import {
+  CandidateChoseToProceedWithTestInWelsh,
+  CandidateChoseToProceedWithTestInEnglish,
+} from '../../../modules/tests/communication-preferences/communication-preferences.actions';
+import {
+  getCommunicationPreference,
+} from '../../../modules/tests/communication-preferences/communication-preferences.reducer';
+import {
+  getConductedLanguage,
+} from '../../../modules/tests/communication-preferences/communication-preferences.selector';
+import { AuthenticationProvider } from '../../../providers/authentication/authentication';
+import { BasePageComponent } from '../../../shared/classes/base-page';
+import { GearboxCategory } from '@dvsa/mes-test-schema/categories/Common';
+import { TestCategory } from '@dvsa/mes-test-schema/categories/common/test-category';
+import { PASS_CERTIFICATE_NUMBER_CTRL } from '../components/pass-certificate-number/pass-certificate-number.constants';
+
+interface PassFinalisationPageState {
+  candidateName$: Observable<string>;
+  candidateUntitledName$: Observable<string>;
+  candidateDriverNumber$: Observable<string>;
+  testOutcomeText$: Observable<string>;
+  applicationNumber$: Observable<string>;
+  provisionalLicenseProvidedRadioChecked$: Observable<boolean>;
+  provisionalLicenseNotProvidedRadioChecked$: Observable<boolean>;
+  passCertificateNumber$: Observable<string>;
+  transmission$: Observable<GearboxCategory>;
+  transmissionAutomaticRadioChecked$: Observable<boolean>;
+  transmissionManualRadioChecked$: Observable<boolean>;
+  d255$: Observable<boolean>;
+  debriefWitnessed$: Observable<boolean>;
+  conductedLanguage$: Observable<string>;
+}
+
+@IonicPage()
+@Component({
+  selector: '.pass-finalisation-cat-c-page',
+  templateUrl: 'pass-finalisation.cat-c.page.html',
+})
+export class PassFinalisationCatCPage extends BasePageComponent {
+  pageState: PassFinalisationPageState;
+  passCertificateCtrl: string = PASS_CERTIFICATE_NUMBER_CTRL;
+  @ViewChild('passCertificateNumberInput')
+  passCertificateNumberInput: ElementRef;
+  inputSubscriptions: Subscription[] = [];
```

Comment:
This subscription is never `subscribe()`'d to.

Reply:
Not sure what to do with this right now. Let's revisit at another time